src/Nethermind/Nethermind.Consensus/Requests/ConsolidationRequestProcessor.cs

```diff
@@ -0,0 +1,66 @@
+// SPDX-FileCopyrightText: 2024 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Nethermind.Core;
+using Nethermind.Core.ConsensusRequests;
+using Nethermind.Core.Crypto;
+using Nethermind.Core.Extensions;
+using Nethermind.Core.Specs;
+using Nethermind.Crypto;
+using Nethermind.Evm;
+using Nethermind.Evm.Tracing;
+using Nethermind.Evm.TransactionProcessing;
+using Nethermind.Int256;
+using Nethermind.State;
+
+namespace Nethermind.Consensus.Requests;
+
+// https://eips.ethereum.org/EIPS/eip-7251#block-processing
```

Comment:
Where are we performing the check that the request list matches the result of the smart contract call? (see: https://github.com/ethereum/EIPs/blob/133576571773222705b93372fc9cad7f2a17f39e/EIPS/eip-7251.md?plain=1#L540) (maybe not implemented yet?)